packages/core/src/Serialization/Certificates/PoolRegistration.ts

```diff
@@ -170,7 +174,7 @@ export class PoolRegistration {
    * @param cert core PoolRegistrationCertificate object.
    */
   static fromCore(cert: Cardano.PoolRegistrationCertificate) {
-    return new PoolRegistration(PoolParams.fromCore(cert.poolParameters)); // TODO: Core type does not support script hash as credential, fix?
+    return new PoolRegistration(PoolParams.fromCore(cert.poolParameters), cert.deposit); // TODO: Core type does not support script hash as credential, fix?
```

Comment:
```suggestion
    return new PoolRegistration(PoolParams.fromCore(cert.poolParameters), cert.deposit); 
```
This was not introduced by this PR, but I noticed this TODO makes no sense here and is probably a result of a bad copy/paste 😅

Reply:
Indeed I had some difficulties understanding its meaning...
If there are no objections, I'll remove it

Reply:
8a790b0aef should solve this